--- plugins/gtkui/plcommon.c.org	2015-09-14 03:12:34.407010054 +0700
+++ plugins/gtkui/plcommon.c	2015-09-14 03:14:09.894014371 +0700
@@ -1313,7 +1313,7 @@
         inf->format = strdup ("%length%");
         break;
     case 8:
-        inf->format = strdup ("%track%");
+        inf->format = strdup ("%track number%");
         break;
     case 9:
         inf->format = strdup ("$if(%album artist%,%album artist%,Unknown Artist)");
