--- deadbeef-devel/Makefile.am	2014-08-03 17:30:39.863390242 +0400
+++ deadbeef-devel/Makefile.am	2014-08-03 17:30:45.503390323 +0400
@@ -44,7 +44,7 @@
 ddb_misc_filebrowser_GTK3_la_SOURCES = $(filebrowser_SOURCES)
 ddb_misc_filebrowser_GTK3_la_LDFLAGS = -module
 ddb_misc_filebrowser_GTK3_la_LIBADD  = $(LDADD) $(GTK3_DEPS_LIBS)
-ddb_misc_filebrowser_GTK3_la_CFLAGS  = -std=c99 $(GTK3_DEPS_CFLAGS) -Wall -Werror -g
+ddb_misc_filebrowser_GTK3_la_CFLAGS  = -std=c99 $(GTK3_DEPS_CFLAGS) -Wall -g
 endif
 
 docsdir = $(docdir)
--- deadbeef-devel/Makefile.in	2014-08-03 17:31:58.343391362 +0400
+++ deadbeef-devel/Makefile.in	2014-08-03 17:32:04.800058120 +0400
@@ -405,7 +405,7 @@
 @HAVE_GTK3_TRUE@ddb_misc_filebrowser_GTK3_la_SOURCES = $(filebrowser_SOURCES)
 @HAVE_GTK3_TRUE@ddb_misc_filebrowser_GTK3_la_LDFLAGS = -module
 @HAVE_GTK3_TRUE@ddb_misc_filebrowser_GTK3_la_LIBADD = $(LDADD) $(GTK3_DEPS_LIBS)
-@HAVE_GTK3_TRUE@ddb_misc_filebrowser_GTK3_la_CFLAGS = -std=c99 $(GTK3_DEPS_CFLAGS) -Wall -Werror -g
+@HAVE_GTK3_TRUE@ddb_misc_filebrowser_GTK3_la_CFLAGS = -std=c99 $(GTK3_DEPS_CFLAGS) -Wall -g
 docsdir = $(docdir)
 docs_DATA = README COPYING AUTHORS ChangeLog
 dist_data_DATA = 
