--- Makefile.am.org	2016-01-27 00:45:52.979845368 +0700
+++ Makefile.am	2016-01-27 00:46:07.159845166 +0700
@@ -46,13 +46,13 @@
 
 if HAVE_GTK2
 ddb_misc_filebrowser_GTK2_la_SOURCES = $(filebrowser_SOURCES)
-ddb_misc_filebrowser_GTK2_la_LDFLAGS = -module
+ddb_misc_filebrowser_GTK2_la_LDFLAGS = -module -avoid-version
 ddb_misc_filebrowser_GTK2_la_LIBADD  = $(LDADD) $(GTK2_DEPS_LIBS)
 ddb_misc_filebrowser_GTK2_la_CFLAGS  = -std=c99 $(GTK2_DEPS_CFLAGS) $(EXTRA_FLAGS) $(VERSION_FLAGS)
 endif
 if HAVE_GTK3
 ddb_misc_filebrowser_GTK3_la_SOURCES = $(filebrowser_SOURCES)
-ddb_misc_filebrowser_GTK3_la_LDFLAGS = -module
+ddb_misc_filebrowser_GTK3_la_LDFLAGS = -module -avoid-version
 ddb_misc_filebrowser_GTK3_la_LIBADD  = $(LDADD) $(GTK3_DEPS_LIBS)
 ddb_misc_filebrowser_GTK3_la_CFLAGS  = -std=c99 $(GTK3_DEPS_CFLAGS) $(EXTRA_FLAGS) $(VERSION_FLAGS)
 endif
